Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerWithBypass.mo

```diff
@@ -0,0 +1,397 @@
+  parameter Modelica.SIunits.Time Td(
+    final min=0)=0.1
+    "Derivative time constant of the tower PID controller"
+    annotation (Dialog(enable=(controllerType == Modelica.Blocks.Types.SimpleController.PD or controllerType == Modelica.Blocks.Types.SimpleController.PID),group="Control Settings"));
+  Medium.ThermodynamicState sta_a=Medium.setState_phX(
+    port_a.p,
+    noEvent(
+      actualStream(
+        port_a.h_outflow)),
+    noEvent(
+      actualStream(
+        port_a.Xi_outflow))) if show_T
+    "Medium properties in port_a";
+  Medium.ThermodynamicState sta_b=Medium.setState_phX(
+    port_b.p,
+    noEvent(
+      actualStream(
+        port_b.h_outflow)),
+    noEvent(
+      actualStream(
+        port_b.Xi_outflow))) if show_T
+    "Medium properties in port_b";
+  Modelica.Fluid.Interfaces.FluidPort_a port_a(
+    redeclare package Medium=Medium)
+    "Fluid connector a (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{-110,-10},{-90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b(
+    redeclare package Medium=Medium)
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{90,-10},{110,10}})));
+  Modelica.Blocks.Interfaces.BooleanInput on[num]
+    "On signal for cooling towers"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}})));
+  Modelica.Blocks.Interfaces.RealInput TWetBul(
+    final unit="K",
+    displayUnit="degC")
+    "Entering air wetbulb temperature"
+    annotation (Placement(transformation(extent={{-140,-40},{-100,0}})));
+  Modelica.Blocks.Interfaces.RealOutput PFan[num](
+    each final quantity="Power",
+    each final unit="W")
+    "Electric power consumed by fan"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Modelica.Blocks.Interfaces.RealOutput TLvg[num](
+    each final unit="K",
+    each displayUnit="degC")
+    "Leaving water temperature"
+    annotation (Placement(transformation(extent={{100,20},{120,40}})));
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerParellel cooTowSys(
+    final use_inputFilter=use_inputFilter,
+    redeclare final package Medium=Medium,
+    final num=num,
+    final show_T=show_T,
+    final m_flow_nominal=m_flow_nominal/num,
+    final dp_nominal=dp_nominal,
+    final ratWatAir_nominal=ratWatAir_nominal,
+    final TAirInWB_nominal=TAirInWB_nominal,
+    final TWatIn_nominal=TWatIn_nominal,
+    final dT_nominal=dT_nominal,
+    final PFan_nominal=PFan_nominal,
+    final energyDynamics=energyDynamics)
+    "Cooling tower system"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage valByp(
+    redeclare final package Medium=Medium,
+    final m_flow_nominal=m_flow_nominal*0.0001,
+    final dpValve_nominal=dp_nominal,
```

Comment:
`dp_nominal` is presumably for the cooling tower system (it will be much larger than a typical valve value). Please propagate a parameter `dpValve_nominal`, which can be used for both the bypass valve and isolation valves within `coTowSys`